src/dedicated.njk

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
@Yndira-E @Hasmin-AC I thought this was supposed to be under `/product`? So `flowfuse.com/product/dedicated`?

Reply:
Hi @ZJvandeWeg, according to [the briefing](https://docs.google.com/document/d/1pbNz2JlKwB7GbkUemjvA4gF_qjImV360PXUl6Dj-Tos/edit?usp=sharing) , the specified URL is `flowfuse.com/dedicated`

Reply:
And I didn’t notice any mention of adding it to the main navigation. Should I proceed to move it and include it under the product menu, as you suggested?

Reply:
In our website discussion we decided that it'll be part of pricing page (similar to gitlab) as it's a type of deployment. 
So not listed under product.